src/cr/cube/cube.py

```diff
@@ -428,7 +442,7 @@ def title(self):
         return self._cube_dict["result"].get("title", "Untitled")
 
     @lazyproperty
-    def unweighted_counts(self):
+    def unweighted_counts(self) -> "np.ndarray[np.int64]":
```

Comment:
also here why type is in quotes?

Reply:
because `np.ndarray[np.int64]` fails with the error about the `type` can't be initialized or something...